boilerplate/openshift/golang-osd-operator/build-opm-catalog.sh

```diff
@@ -286,6 +286,9 @@ function add_current_version_to_bundle_versions_file() {
 
     git commit -m "$message"
```

Comment:
Seems like this should be wrapped with a `[[ "$DRY_RUN" == "false" ]]` as well?

Reply:
No need to, because that is done locally in a temporal folder... I left it there so that you people can inspect how things would be pushed

Reply:
Sounds good, thanks!